### PR TITLE
Bump deps for 8.3.0 

### PR DESCRIPTION
**Pull request by ncooke3**

### CODE REVIEW

**Review by paulb777 — APPROVED** [review]
LGTM if the only failure is the Crashlytics quickstart

**Comment by paulb777** [comment]
Merged for rest of tonight's nightlies

**Comment by paulb777** [comment]
Tags updated after merge

**Comment by ncooke3** [comment]
thanks @paulb777!

**Pull request merged**
